src/main/kotlin/org/rust/lang/core/RsPsiPattern.kt

```diff
@@ -197,6 +197,14 @@ object RsPsiPattern {
                 eq?.elementType == EQ && feature?.textMatches("feature") == true
             }
 
+    val insideCrateTypeAttrValue: PsiElementPattern.Capture<PsiElement> = psiElement(STRING_LITERAL)
```

Comment:
Now we have duplicate code here and in `RsUnknownCrateTypesInspection`. Maybe it's worth using this pattern in the inspection as well?